test/Subdomain.spec.js

```diff
@@ -104,7 +104,7 @@ describe('DCL Names V2', function() {
 
     creationParams = {
       ...fromDeployer,
-      gas: 6e6,
+      //gas: 6e6,
```

Comment:
Coverage needs to set this value higher. Setting it here overrides the custom provider settings and causes deployments to run out of gas. 

If you wanted to keep `6e6` as an upper bound in the conventional test run I believe you could set that in the BuidlerEVM options.